src/main/java/com/globant/carrito/client/Clients.java

```diff
@@ -0,0 +1,130 @@
+package com.globant.carrito.client;
+
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+import com.globant.carrito.cart.Carts;
+
+@Entity
+public class Clients {
+
+	@Id
+	@GeneratedValue
+	private int clientId;
+	
+	@Column
+	private String name;
+	
+	@Column
+	private String userName;
+	
+	@Column
+	private String password;
+	
+	@Column
+	private String shippingAddress;
+	
+	@Column
+	private String telephone;
+	
+	@Column
+	private String email;
+	
+	@Column
+	private boolean mailist;
+	
+	@OneToMany(cascade = CascadeType.ALL, mappedBy = "Clients")

```

Comment:
Así es, lo cambié a "client".

Date: Wed, 4 Feb 2015 06:59:43 -0800
From: notifications@github.com
To: cart_client@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [cart_client] Avance del shopping cart hasta el 03/02/15. (#1)

In src/main/java/com/globant/carrito/client/Clients.java:

> +	@Column
> +	private String password;
> +	
> +	@Column
> +	private String shippingAddress;
> +	
> +	@Column
> +	private String telephone;
> +	
> +	@Column
> +	private String email;
> +	
> +	@Column
> +	private boolean mailist;
> +	
> +	@OneToMany(cascade = CascadeType.ALL, mappedBy = "Clients")

Aca en mappedBy va el nombre de la variable que se usa para hacer la relación inversa. En este caso, la variable está en Carts en la línea 28. Se llama clientId, pero por el comentario que te puse probablemente cambie a client o algo asi.


—
Reply to this email directly or view it on GitHub.